src/Shouldly.Tests/ConventionTests/ShouldlyConventions.ShouldHaveCustomMessageOverloads.approved.txt

```diff
@@ -0,0 +1,10 @@
+﻿'The following shouldly methods are missing one or more of the custom message overloads' for 'Types in Shouldly extension classes'
+----------------------------------------------------------------------------------------------------------------------------------
+
+	ShouldMatchApproved(this System.String actual)
```

Comment:
These methods all have strange overloads with optional params etc. Which is why they are approved failures.